themes/socialbase/templates/layout/page.html.twig

```diff
@@ -63,14 +67,22 @@
         {{ page.title }}
       {% endif %}
 
+      {% if style == 'sky' %}
+        {% if page.secondary_navigation %}
+          {{ page.secondary_navigation }}
+        {% endif %}
+      {% endif %}
+
       {% if page.complementary_top or page.complementary_bottom %}
         <aside class="region--complementary" role="complementary">
-          {% if page.complementary_top %}
-            {{ page.complementary_top }}
-          {% endif %}
-          {% if page.complementary_bottom %}
-            {{ page.complementary_bottom }}
-          {% endif %}
+          <div class="complementary--content-merged card">
```

Comment:
Does this `div` truly add value here? Can we not just style the `region--complementary` instead?